src/handbook/product/strategy.md

```diff
@@ -4,93 +4,67 @@ navTitle: Strategy
 
 # Product Strategy
 
-Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowFuse’s vision, to become the leading "Low-Code IIoT Application Platform".
-FlowFuse will allow everyone to build custom applications to achieve their business goals and depend less on off-the-shelf software.
-It’s a draft, as everything in the handbook is a draft, so we will revisit regularly and iterate when there is a reason to do so. FlowFuse is a 100 percent agile company, so our product is; timing is everything.
-Have a look at our [roadmap](/product/roadmap/) for what we're pursuing in the upcoming releases.
+Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowFuse’s [vision and strategy](..//company/strategy.md). FlowFuse will allow everyone to build custom applications to achieve their business goals and depend less on off-the-shelf software.  It is a living strategy, so we will revisit on a regular basis and iterate when there is reason to do so; timing is everything. 
 
-### Advancing Enterprise-Readiness
+## Market Vision
 
-FlowFuse is committed to augmenting the enterprise-readiness of Node-RED. Our strategy emphasises enhancing audit and compliance capabilities, instituting stringent preventive and corrective controls, and bolstering security. By integrating these vital aspects, we aim to create a reliable, secure, and scalable environment. This initiative underlines our dedication to surpassing enterprise expectations and strengthening Node-RED's position in the industry.
+**Our strategic goal is to be the leading "Low-Code IIoT Application Platform"**
 
-### Node-RED Deployments at the Edge (Device Management)
+An IIoT platform is a comprehensive solution that securely collects, integrates, and manages vast volumes of data from IoT endpoints. It's the bridge between Information Technology (IT) and Operational Technology (OT), bringing about a seamless fusion of diverse protocols and data formats, facilitating edge and cloud analytics, and enhancing overall asset management. This digital conduit not only augments traditional OT functionalities but also propels sectors like manufacturing and energy towards a more digitized, efficient future. A robust IIoT platform encapsulates essential elements such as unwavering security, sophisticated automation, and a commitment to sustainability. It fosters an environment where interoperability with a myriad of business applications is not just a feature but a foundational aspect. From meticulous device management and seamless integration to advanced data analytics and application development (see also [IIoT platform definition](https://www.gartner.com/doc/reprints?id=1-2C757S9J&ct=230105&st=sb)).
```

Comment:
My take:
- It's harder to innovate at the pace required to extract ROI
- Development of custom solutions is expensive, because 1) it requires lot of people to be involved with different skill sets 2) hard to capture data from the different sources and make it accessable for the org
   - And so traditional industries select point solutions, leaving ROI on the table
- If all data, regardless of where it was produced or needed to be comsumed, was available and useful for _all_ engineers in the org (mechenical, elektrical) increased ROI solutions can be build and maintained by mid to large sized companies in traditional industries.

Reply:
@knolleary I appreciate your thoughts here, I'm still not sure if we should want to be an IIoT Platform, or set another target on the horizon?

Reply:
(The gartner link in the text has expired)

I've been trying to come up with a response that helps move this conversation forward... and struggling to find the right words, so this may be a bit rambling and perhaps simplistic.

This links to our [Company Strategy](https://flowfuse.com/handbook/company/strategy/) but I don't see it reference any specific points of the strategy. That makes it hard to see how the two relate.

The company strategy lists the following goals:

 - Drive the growth and success of the Node-RED project
 - Become the standard way of running any Node-RED for professional usage
 - Drive adoption of Node-RED in traditional industries

From a product strategy point of view, the question we have to answer is how each item in this doc relates back to those goals.

The same company strategy leans into the association with Node-RED - we don't currently justify the value Node-RED can bring, we justify the value FlowFuse can bring to existing Node-RED users. A basic question is then, does our vision (for the next 2 years) look beyond that, or is that implicitly what is at the heart of the vision.

We want FlowFuse to be the obvious choice when working with Node-RED in the enterprise.

My hesitation to setting our product strategy to be an IIoT Platform is whether that goes beyond the Node-RED-focus of our existing strategy. But then, I don't have a good label for it if we don't call it a low-code IIoT Platform.

The more I try to get this into words, the more I think we need to just set aside a couple hours and discuss this in real-time.

Reply:
As discussed in our meeting: 
[Update strategy.md](https://github.com/FlowFuse/website/pull/1650/commits/f9e17c44551445ff782c77de5baba911b767e4af)